Tools/TestWebKitAPI/Tests/WebKitCocoa/ProcessSwapOnNavigation.mm

```diff
@@ -7946,6 +7946,7 @@ static void configureLockdownWKWebViewConfiguration(WKWebViewConfiguration *conf
     [config.preferences _setMediaDevicesEnabled:YES];
     config.preferences._mediaCaptureRequiresSecureConnection = NO;
     [config.preferences _setNotificationsEnabled:YES];
+    [config.preferences _setPushAPIEnabled: YES];
```

Comment:
I don't think this should be enabled by default for PSON tests. We explicitly call `_setPushAPIEnabled:` in the few tests where we actually want to test push API (and in those tests we launch a test-only `webpushd` daemon).

Reply:
This is actually a Lockdown Mode test, to confirm that even if someone turns a feature on in the preferences, it will be automatically turned off in the Lockdown version of the process.

Reply:
Sounds good then.